src/plugin.js

```diff
@@ -162,7 +163,15 @@ const perSourceBehaviors = function() {
       // Make sure the wrapped listener and the original listener share a GUID,
       // so that video.js properly removes event bindings when `off()` is passed
       // the original listener!
-      wrappedListener.guid = originalListener.guid;
+      //
+      // This property is used internally by Video.js for tracking bound
+      // listeners in its event system. Normally, Video.js creates this `guid`
+      // through its `bind` method, but that method is deprecated in public
+      // interfaces.
+      //
+      // Basically, this allows us to avoid deprecation warnings from calling
+      // `videojs.bind` when used with Video.js 8.
+      wrappedListener.guid = originalListener.guid = passedListener.guid = `psb-${++psbGuid}`;
```

Comment:
See comments for complete details, but this `guid` is prefixed to avoid any chance of collision with internal `guid`s generated within Video.js.